src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -6529,29 +6529,74 @@ return Popper;
 })));
 //# sourceMappingURL=bootstrap.js.map
 
-function getConfirmRemoveMessage() {
-    return $('#confirmRemoveMessage').data('value');
+function confirmDialog(title, message, handler) {
+    if (title === undefined) {
+        title = $('#confirmRemoveModalMetadata').data('title');
+    }
+
+    if (message === undefined) {
+        message = $('#confirmRemoveModalMetadata').data('message');
+    }
+
+    $(`<div id="confirmRemoveModal" class="modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">`+ title + `</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <p>`+ message + `</p>
+                </div>
+                <div class="modal-footer">
+                    <button id="modalOkButton" type="button" class="btn btn-danger">`+ $('#confirmRemoveModalMetadata').data('ok') + `</button>
+                    <button id="modalCancelButton" type="button" class="btn btn-secondary" data-dismiss="modal">`+ $('#confirmRemoveModalMetadata').data('cancel') + `</button>
+                </div>
+            </div>
+        </div>
+    </div>`).appendTo("body");
```

Comment:
What happens if I click cancel and open the dialog again? Does it add this element multiple times with the same id?